src/backend/src/prisma/seed-data/users.seed.ts

```diff
@@ -17,7 +17,8 @@ const thomasEmrax: Prisma.UserCreateInput = {
       defaultTheme: Theme.DARK,
       slackId: 'emrax'
     }
-  }
+  },
+  favoriteProjects: {}
```

Comment:
should be an array right?

Reply:
Check the new changes, I need to use connect to connect it to a project, or multiple projects, but we do not have any seed data projects so I do not know what to connect it to. Or if I should make some seed projects